Common Utilities/Config.cs

```diff
@@ -16,8 +16,20 @@ public class Config : IConfig
         [Description("Wether or not debug messages should be shown.")]
         public bool Debug { get; set; } = false;
 
-        [Description("Wether or not SCP-049 should be able to talk to humans.")]
-        public bool Scp049Speech { get; set; } = true;
+        [Description("Can 106 speak?")]
```

Comment:
You should use the proper SCP names, not their nicknames for all of these properties. Also the description should specify that it relates to them speaking to humans, otherwise people might assume it refers to auto-muting them.